source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,204 @@
+.. _security:
+
+Clear Linux\* OS Security 
+**********************
+
+Clear Linux\* aims to make systemic and layered security-conscious decisions
```

Comment:
@puneetse. Let's try to integrate the name change here too. 
Please revise first mention to: "Clear Linux\* OS"